php/README.md

```diff
@@ -0,0 +1,51 @@
+# PHP Tracing Example
+
+This example shows a mix of manual and auto-instrumented PHP code.  It uses the
+[Symfony demo app](https://github.com/symfony/demo) to provide an HTTP server
+that the [example.php](./example.php) script makes a request to.
+
+## Running
+
+To run this example, first install the PHP tracer by using your [preferred
+installation method](https://docs.signalfx.com/en/latest/apm/apm-instrument/apm-php.html).
+Note that this must be done on in a Linux environment, as MacOS is not
+supported.  If you do not wish to install the tracer yourself, you can use the
+Dockerfile provided in this directory.
+
+**If you do not have the [Smart Agent deployed on the same
+host](https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-agent.html)**,
+you can configure the PHP tracer to send directly to our global ingest
+server by setting the following envvars in your shell before running the next
+command:
+
+```sh
+$ export SIGNALFX_ACCESS_TOKEN=<your org token>
+$ export SIGNALFX_ENDPOINT_URL=https://ingest.signalfx.com/v1/trace  # Or the appropriate ingest for your realm
```

Comment:
These should probably be standardized https://github.com/signalfx/signalfx-ruby-tracing/blob/master/lib/signalfx/tracing.rb#L17

Reply:
Probably so, but I don't think the Ruby name of "ingest" is the best.  The Jaeger "endpoint" term seems to be the most generic and clear to cover everything from going to the agent, gateway, and directly to ingest.